backend/src/main/java/com/ody/meeting/domain/Meeting.java

```diff
@@ -51,4 +51,10 @@ public void updateInviteCode(String inviteCode) {
     public LocalDateTime getMeetingTime() {
         return LocalDateTime.of(date, time).withSecond(0).withNano(0);
     }
+
+    public boolean isWithinPast24HoursOrLater() {
+        LocalDateTime dateTime = LocalDateTime.of(date, time);
+        LocalDateTime standard = LocalDateTime.now().minusHours(24).withSecond(0);
+        return dateTime.isEqual(standard) || dateTime.isAfter(standard);
```

Comment:
[제안- 선택]

시간 관련 기준점을 바꾸면 한가지 표현으로 통일할 수 있어요. 그러나, 기준점이 약속일자여야 조금 더 직관적일 수도 있겠다는 생각이에요! 반영은 자유롭게!

```suggestion
        return standard.isBefore(dateTime);
```

Reply:
[반영] [850156f](https://github.com/woowacourse-teams/2024-ody/pull/302/commits/850156ffe2f2c587af451457c2ed29a405c6b51e)